README.md

```diff
@@ -10,18 +10,19 @@ will always override the value provided in the yaml file.
 
 ### Environment variables
 
-| Name                        | Description                              | Type     | Default                                |
-|-----------------------------|------------------------------------------|----------|----------------------------------------|
-| INFINITEWISDOM_BOT_TOKEN                   | The bot token used to authenticate the bot with telegram | String | `-` |
-| INFINITEWISDOM_MAX_URL_POOL_SIZE           | Maximum number of URLs to keep in the pool | Integer | `10000` |
-| INFINITEWISDOM_IMAGE_POLLING_TIMEOUT       | Timeout in seconds between image api requests | Integer | `1` |
-| INFINITEWISDOM_GREETING_MESSAGE            | Specifies the message a new user is greeted with | String | `Send /inspire for more inspiration :) Or use @InfiniteWisdomBot in a group chat and select one of the suggestions.` |
-| INFINITEWISDOM_INLINE_BADGE_SIZE           | Number of items to return in a single inline request badge | Integer | `16` |
-| INFINITEWISDOM_PERSISTENCE_TYPE            | Type of persistence to use | String | `local` |
-| INFINITEWISDOM_PERSISTENCE_PATH            | Path of pickle persistence | String | `/tmp/infinitewisdom.pickle` |
-| INFINITEWISDOM_PERSISTENCE_URL             | URL to use for SQLAlchemy connection | String | `sqlite:////tmp/infinitewisdom.db` |
-| INFINITEWISDOM_IMAGE_ANALYSIS_TYPE         | Type of image analysis to use | String | `None` |
-| INFINITEWISDOM_IMAGE_ANALYSIS_AUTH_FILE    | Path of Google Vision auth file | String | `None` |
+| Name                                                  | Description                              | Type     | Default                                |
+|-------------------------------------------------------|------------------------------------------|----------|----------------------------------------|
+| INFINITEWISDOM_TELEGRAM_BOT_TOKEN                     | The bot token used to authenticate the bot with telegram | `str` | `-` |
+| INFINITEWISDOM_TELEGRAM_GREETING_MESSAGE              | Specifies the message a new user is greeted with | `str` | `Send /inspire for more inspiration :) Or use @InfiniteWisdomBot in a group chat and select one of the suggestions.` |
+| INFINITEWISDOM_TELEGRAM_INLINE_BADGE_SIZE             | Number of items to return in a single inline request badge | `int` | `16` |
+| INFINITEWISDOM_CRAWLER_TIMEOUT                        | Timeout in seconds between image api requests | `float` | `1` |
+| INFINITEWISDOM_PERSISTENCE_TYPE                       | Type of persistence to use | `str` | `local` |
```

Comment:
Is it really `local`? I thought `pickle` or `sql`?

Reply:
Correct! Currently the default is pickle but since it will be removed eventually I don't see why we shouldn't use sql as the default right away. I will change this.